---
title:Software Tools
layout: software
---

<h2>Glossary for Software Tools</h2>
<h3>Software type</h3>
<ul>
<li><strong>Stand-alone software</strong><br>
Software that is downloaded and installed on a computer.</li><br>
<li><strong>Web application</strong><br> 	
Application accessed and used using a web browser.</li><br>
<li><strong>Web service</strong><br>	
Application invoked over the internet by computational means. It accepts machine readable data and provides remachine readable data as result.</li>
</ul>

<h3>Data types</h3>
<ul>
<li><strong>MS</strong><br>
Mass spectrometric data</li>
</ul><br>

<h2>Software Tools</h2>
<h3>GRITS Toolbox</h3>
GRITS Toolbox is a glycomics MS data processing software. The software allows not just to load, visualize and browser glycomics MS data but also to annotate the data with glycan structures and their fragments. Automatic annotated data can be manually evaluated, changed and exported to Excel. In addition side by side comparison of the different experiment for the study of glycosylation changes and glycosylation expression changes can be performed as well. In addition the software also captures and manages meta data such as sample description and description of the performed experiment.<br> 
<table>
<tr>
<td><strong>Processed data</strong></td>
<td>MS</td>
</tr>
<tr>
<td><strong>Software type</strong></td>
<td>Standalone software</td>
</tr>
<tr>
<td><strong>Availability</strong></td>
<td><a href="http://www.grits-toolbox.org/">http://www.grits-toolbox.org/</a></td>
</tr>
<tr>
<td><strong>License
<td>none</td>
</tr>
<tr>
<td><strong>Status</strong></td>
<td>stable, continues development</td>
</tr>
<tr>
<td><strong>Funding</strong></td>
<td>NIH</td>
</tr>
<tr>
<td><strong>Future plans</strong></td>
<td>quantification of glycomics data; CMS like methods for glycomics; integration of other data types, such as qRT-PCR</td>
</tr>
<tr>
<td><strong>Contact</strong></td>
<td>Rene Ranzinger:&nbsp;<a href="mailto:rene@ccrc.uga.edu">rene@ccrc.uga.edu</a></td>
</tr>
</table>

<h3>GODESS</h3>
NMR spectrum simulation service for carbohydrate-containing molecules (including polymers and glycoconjugates). The output includes signal assignment tables, 1D and multiple 2D spectra. For every simulated chemical shift the predictor reports the expected error, trustworthiness metrics and databases references to data used in the simulation. The software uses empirical approach with own dedicated databases, statistical approach based on CSDB content, and hybrid approach. Supported nuclei are 13C and 1H. Solvent and temperature are taken into account. On the level of structures it supports most structural features of saccharides (several hundred residues, alditols, furanoses, amino acids, fatty acids, sphingoids; glycosidic, amide, or diester linkages). Average accuracy obtained on a large pool of structures was 0.86 ppm for &sup1;&sup3;C and 0.07ppm for &sup1;H.<br>
<table>
<tr>
<td><strong>Processed data</strong></td>
<td>NMR, structures</td>
</tr>
<tr>
<td><strong>Software type</strong></td>
<td>web application</td>
</tr>
<tr>
<td><strong>Availability</strong></td>
<td><a href="http://csdb.glycoscience.ru/database/core/nmrsim.html">http://csdb.glycoscience.ru/database/core/nmrsim.html</a></td>
</tr>
<tr>
<td><strong>License</strong></td>
<td>none (free)</td>
</tr>
<tr>
<td><strong>Status</strong></td>
<td>stable, continues development</td>
</tr>
<tr>
<td><strong>Funding</strong></td>
<td>Russiann Foundation for Basic Research</td>
</tr>
<tr>
<td><strong>Future plans</strong></td>
<td>HMBC prediction, structure iterator & ranking</td>
</tr>
<tr>
<td><strong>Contact</strong></td>
<td>Phyl Toukach:&nbsp;<a href="mailto:netbox@toukach.ru">netbox@toukach.ru</a></td>
</tr>
<tr>
</table>

<h3>GlycoDigest</h3>
GlycoDigest is a tool that simulates the action of exoglycosidases on released oligosaccharides. It has been developed to assist glycobiologists design mixtures of exoglycosidases that can be used to guide the precise determination of glycan structures.<br>
<table>
<tr>
<td><strong>Processed data</strong></td>
<td>glycan structures</td>
</tr>
<tr>
<td><strong>Software type</strong></td>
<td>Standalone software and integrated in GlycoBase and UniCarbKB</td>
</tr>
<tr>
<td><strong>Availability</strong></td>
<td><a href="http://www.glycodigest.org">http://www.glycodigest.org</a></td>
</tr>
<tr>
<td><strong>License</strong></td>
<td>CC</td>
</tr>
<tr>
<td><strong>Status</strong></td>
<td>stable, continues development</td>
</tr>
<tr>
<td><strong>Funding</strong></td>
<td>SIB; National eResearch Collaboration Tools and Resources (NeCTAR) Project</td>
</tr>
<tr>
<td><strong>Future plans</strong></td>
<td>Extend range of enzymes for simulation of activity; budding plan with CAZy team.</td>
</tr>
<tr>
<td><strong>Contact</strong></td>
<td>Matthew Campbell:&nbsp;<a href="mailto:matthew.campbell@mq.edu.au">matthew.campbell@mq.edu.au</a><br>
Frederique Lisacek:&nbsp;<a href="frederique.lisacek@isb-sib.ch">frederique.lisacek@isb-sib.ch</a></td>
</tr>
</table>

<h3>3D structure validation tools (CARP, pdb-care)</h3>
Tools for validation of carbohydrate 3D structure data.<br> 
<table>
<tr>
<td><strong>Processed data</strong></td>
<td>PDB-formatted 3D structures (carbohydrates, glycoproteins, protein-carbohydrate complexes)</td>
</tr>
<tr>
<td><strong>Software type</strong></td>
<td>Web application, Integrated in Glycosciences.de
</tr>
<tr>
<td><strong>Availability</strong></td>
<td><a href="http://www.glycosciences.de/tools/">http://www.glycosciences.de/tools/</a></td>
</tr>
<tr>
<td><strong>License</strong></td>
<td>	</td>	
</tr>
<tr>
<td><strong>Status</strong></td>
<td>stable, continues development</td>
</tr>
<tr>
<td><strong>Funding</strong></td>
<td>-</td>
</tr>
<tr>
<td><strong>Future plans</strong></td>
<td>Include MonosaccharideDB routines for improved handling of modified residues.</td>
</tr>
<tr>
<td><strong>Contact</strong></td>
<td>Thomas Lütteke:&nbsp;<a href="mailto:thomas.luetteke@vetmed.uni-giessen.de">thomas.luetteke@vetmed.uni-giessen.de</a></td>
</tr>
</table>

<h3>Sweet-II</h3>
Building of 3D structure models of carbohydrates.<br> 
<table>
<tr>
<td><strong>Processed data</strong></td>
<td>carbohydrate “sequence” (2D-structure)</td>
</tr>
<tr>
<td><strong>Software type</strong></td>
<td>Web application, Integrated in Glycosciences.de</td>
</tr>
<tr>
<td><strong>Availability</strong></td>
<td><a href="http://www.glycosciences.de/modeling/sweet2/">http://www.glycosciences.de/modeling/sweet2/</a></td>
</tr>
<tr>
<td><strong>License</strong></td>
<td></td>
</tr>
<tr>
<td><strong>Status</strong></td>
<td>stable</td>
</tr>
<tr>
<td><strong>Funding</strong></td>
<td>-</td>
</tr>
<tr>
<td><strong>Future plans</strong></td>
<td>-</td>
</tr>
<tr>
<td><strong>Contact</strong></td>
<td>Thomas Lütteke:&nbsp;<a href="mailto:thomas.luetteke@vetmed.uni-giessen.de">thomas.luetteke@vetmed.uni-giessen.de</a></td>
</tr>
</table>

<h3>GlyProt</h3>
In-silico glycosylation of protein 3D structures.<br> 
<table>
<tr>
<td><strong>Processed data</strong></td>
<td>Protein 3D structure (PDB format), carbohydrate “sequence” (2D-structure)</td>
</tr>
<tr>
<td><strong>Software type</strong></td>
<td>Web application, Integrated in Glycosciences.de</td>
</tr>
<tr>
<td><strong>Availability</strong></td>
<td><a href="http://www.glycosciences.de/modeling/glyprot/">http://www.glycosciences.de/modeling/glyprot/</a></td>
</tr>
<tr>
<td><strong>License</strong></td>
<td></td>
</tr>
<tr>
<td><strong>Status</strong></td>
<td>stable</td>
</tr>
<tr>
<td><strong>Funding</strong></td>
<td>-</td>
</tr>
<tr>
<td><strong>Future plans</strong></td>
<td>-</td>
</tr>
<tr>
<td><strong>Contact</strong></td>
<td>Thomas Lütteke:&nbsp;<a href="mailto:thomas.luetteke@vetmed.uni-giessen.de">thomas.luetteke@vetmed.uni-giessen.de</a></td>
</tr>
</table>

<h3>ISOGlyP (Isoform Specific O-Glycosylation Prediction)</h3>
ISOGlyP  analizes and roughly predicts isoform specific sites of mucin O-glycosylation for an entered protein sequence .  Future goals are to add the long range enhancing effects of remote O-glycosylation and the action of multiple transferases.  This work is being performed by a collaboration of Thomas Gerken at Case Western Reserve Univ. (School of Medicine) and  Ming-Ying Leung at the Univ. Texas at El Paso (Professor of Mathematical Sci. and Director Bioinformatics Program).  The Gerken  lab is generating the actual data utilizing a series of unique random (glyco)peptides (refs below) and the Leung lab has developed the web site.  We presently have the peptide sequence motif/propensity data for 10 of the 20 isoforms on the site and have generated the remote glycopeptide data on ~11 isoforms (not yet on the site).  We also have plans for developing similar approaches to predict the Core 1, Core 3 and sialylation substitution of the peptide GalNAc by similar methods.<br><br> 
References:<br> 
Perrine et al (2009) <a href="http://glycob.oxfordjournals.org/content/19/3/321">http://glycob.oxfordjournals.org/content/19/3/321</a><br>
Gerken et al. (2011) <a href="http://www.jbc.org/content/286/16/14493">http://www.jbc.org/content/286/16/14493</a><br> 
Gerken et al  (2013) <a href="http://www.jbc.org/content/288/27/19900">http://www.jbc.org/content/288/27/19900</a><br> 
Kong et al (2015)  <a href="http://glycob.oxfordjournals.org/content/25/1/55.abstract">http://glycob.oxfordjournals.org/content/25/1/55.abstract</a><br>
<table>
<tr>
<td><strong>Processed Data</strong></td>
<td>FASTA format protein sequence (manual or file input)</td> 
</tr>
<tr>
<td><strong>Software type</strong></td>
<td>web application</td>  
</tr>
<tr>
<td><strong>Availability</strong></td>
<td><a href="http://isoglyp.utep.edu/">http://isoglyp.utep.edu/</a></td>
</tr>
<tr>
<td><strong>License</strong></td>
<td>none</td>
</tr>
<tr>
<td><strong>Status</strong></td>
<td>available, under development</td>
</tr>
<tr>
<td><strong>Funding</strong></td>
<td>NIH (data acquisition and web site 1/2015-11/2018)</td>
</tr>
<tr>
<td><strong>Future plans</strong></td>
<td>include remote glycosylation with improved output graphics, add 		core 1 & 3 and sialylation predictions</td>
</tr>
<tr>
<td><strong>Contacts</strong></td>
<td>T. Gerken: <a href="mailto:txg2@cwru.edu">txg2@cwru.edu</a>&nbsp;Tel:&nbsp;216-368-4556<br>
M. Y. Leung: <a href="mailto:mleung@utep.edu">mleung@utep.edu</a>&nbsp;Tel:&nbsp;915-747-6836</td>
</tr>
</table>






